lib/_stream_writable.js

```diff
@@ -700,20 +670,6 @@ function finish(stream, state) {
   }
 }
 
-function onCorkedFinish(corkReq, state, err) {
-  let entry = corkReq.entry;
-  corkReq.entry = null;
-  while (entry) {
-    const cb = entry.callback;
-    state.pendingcb--;
-    cb(err);
-    entry = entry.next;
-  }
-
-  // Reuse the free corkReq.
-  state.corkedRequestsFree.next = corkReq;
-}
-
```

Comment:
none of this cork stuff is relevant anymore?

Reply:
nope